Week1/homework/js-exercises/9-logTheRemainder.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/*
+1. If `x` equals 7, and the only other statement is `x = x % 3`, what would be the value of `x` after the calculation?
+2. If `y` equals 21, and the only other statement is `y = y % 4`, what would be the value of `y` after the calculation?
+3. If `z` equals 13, and the only other statement is `z = z % 2`, what would be the value of `z` after the calculation?
+*/
+
+var x = 7;
```

Comment:
Do you think it's better to use `var` or `let` here?

Reply:
Here I can use const.
 No harm in that